docs/permissions.md

```diff
@@ -9,3 +9,18 @@ Additionally, the OpenSAFELY framework requires a user to be a member of the `op
 While your repository is private, you may need to add new collaborators who are not named in your original application. Any new collaborators have the same information governance obligations as the original study authors, and must therefore sign the same access agreements; please send any such requests to [team@opensafely.org](mailto:team@opensafely.org).
 
 
+## Project Member Management
+
+When they are approved to use the platform, every OpenSAFELY user is assigned roles that define what they are permitted to do.
+By default, most researchers using the platform will have the ProjectDeveloper role, which permits them to:
+
+* run jobs
+* cancel jobs
+* view job outputs
+* release reviewed outputs
```

Comment:
Is this actually true right now? Not quite, right?

Reply:
as in, I think it's necessary but not sufficient?

Reply:
hah, say you're on the outputs voyage without saying you're on the outputs voyage 😉

Removed until we have that process deployed fully.